editor/src/clj/editor/material.clj

```diff
@@ -361,7 +361,8 @@
 
 (defn- sanitize-sampler [sampler]
   ;; Material$MaterialDesc$Sampler in map format.
-  (dissoc sampler :name-indirections)) ; Only used in built data by the runtime.
+  ;; TODO: The texture field _will_ be used in the editor, just not in this MVP
+  (dissoc sampler :name-indirections :texture :name-hash)) ; Only used in built data by the runtime.
```

Comment:
It seems we're not including `:texture` or `:name-hash` in the `.materialc`  build output produced from the editor yet. Don't we need to implement that before merging?

Reply:
This MVP is only for bob+engine.
Since we cannot load the new content, I don't see a reason to be able to build it?
Next MVP's are for the editor to finish the feature (editing, migration, building)

Reply:
As long as there are no actual issues with it, I prefer checking it in, since:
* We can start using it from a programmers point of view
* The branch doesn't grow old again
* We can split up work

Reply:
I would agree with that assessment when it concerns the `texture` field, but it looks like the `dmRender::GetMaterialSamplerUnit` function relies on a correct `name_hash` in the `.materialc` build output.

See the [old code that has been removed](https://github.com/defold/defold/pull/7380/files#diff-b5b03eb809c81309a369c38a53a3ffc07719f77e8a701bbfae969ca8ec097777L161-L176), and the [new code that calls GetMaterialSamplerUnit](https://github.com/defold/defold/pull/7380/files#diff-b5b03eb809c81309a369c38a53a3ffc07719f77e8a701bbfae969ca8ec097777R235-R244).

Won't it break for existing content if the editor doesn't write it to the build output?

Reply:
Didn't you previously say the "textures" (aka sampler names) was deprecated, and moved to the samplers array?

But no matter.
I guess I'll add the name hash, it will be needed later by the editor anyways.